Dockerfile

```
@@ -1,22 +1,28 @@
-FROM java:7
+FROM ubuntu:20.04 AS builder
 
-ENV RELEASE_DATE 2017-04-23
-ENV H2DATA /h2-data
+ARG RELEASE_DATE=2022-06-13
+ARG RELEASE_VERSION=2.1.214
+
+RUN apt-get update \
+  && apt-get install curl -y \
+  && apt-get install unzip -y \
```

Let's shrink them to one line:
```suggestion
  && apt-get install curl unzip -y \
```